backend/github.js

```diff
@@ -0,0 +1,20 @@
+var exports = module.exports = {};
+var request = require("request");
+
+exports.retrieveOrgs = function (accessToken, callback) {
+  var organisations = [];
+  request({
```

Comment:
Actually, I've been following the structure that has been used in this project. Previous uses of request in this project have been implemented similarly.